src/event-team.html

```diff
@@ -10,21 +10,31 @@ <h1>${team.teamNumber}</h1>
   <p> Maybe add a button to manually remove a match? </p>
 
   <aubs-tabset active.bind='activeTab'>
-    <aubs-tab header='Teams'>
+    <aubs-tab header='Matches'>
       <table class="table table-hover">
         <tr>
           <th>Match Number</th>
           <th>Cube Count</th>
           <th>Foul Count</th>
         </tr>
-        <tr repeat.for="match of matches">
-          <td><a route-href="route.bind:'match-team'; params.bind: {year: 2018, eventCode: match.eventCode, teamNumber: team.teamNumber, matchNumber: match.matchNumber}">${match.matchNumber}</a></td>
+        <tr repeat.for="match of testData">
+          <td><a style="display: block;" route-href="route.bind:'match-team'; params.bind: {year: 2018, eventCode: match.eventCode, teamNumber: team.teamNumber, matchNumber: match.matchNumber}">${match.matchNumber}</a></td>
```

Comment:
since this style is happening in multiple places, it should probably be either a global css rule or a css class